aequilibrae/paths/results/path_results.py

```diff
@@ -60,6 +60,7 @@ def __init__(self) -> None:
         self.origin = None
         self.destination = None
         self.graph: Graph = None
+        self.early_exit = False
```

Comment:
Why do we have both `_early_exit` and `early_exit`?

Reply:
`_early_exit` is intended to privately mark the previous trees state for the path re-computation logic in `update_trace`. `early_exit` is intended to be a public configuration option, say if you want to change a `PathResults` object to use early exit on the next update trace where it previously hadn't or visa versa.

I've also realised I've change this API slightly in the A* PR to be consistent
https://github.com/AequilibraE/aequilibrae/pull/467/files#diff-2639bd28c2998e1d08e05d2939351b7cece776c19fec0eaa9c020f98e2ec33e7L295-R349